src/Sylius/Bundle/ShopBundle/Gulpfile.js

```diff
@@ -23,7 +23,7 @@ var paths = {
         ],
         sass: [
             '../UiBundle/Resources/private/sass/**',
```

Comment:
If you use Shop/Admin/Api, which all depend on Core, I think we won't recommend using them as standalone components and maybe we even remove their composer.json and recommend depending on ``sylius/sylius``, which solves this and many other problems. Thoughts?